packages/core/test/tag-input/tagInputTests.tsx

```diff
@@ -32,19 +26,17 @@ import { Button, Classes, Intent, Keys, Tag, TagInput, TagInputProps } from "../
  */
 const mount = (el: React.ReactElement<TagInputProps>, options?: MountRendererProps) =>
     untypedMount<TagInput>(el, options);
-const shallow = (el: React.ReactElement<TagInputProps>, options?: ShallowRendererProps) =>
-    untypedShallow<TagInput>(el, options);
 
 const VALUES = ["one", "two", "three"];
 
 describe("<TagInput>", () => {
     it("passes inputProps to input element", () => {
         const onBlur = sinon.spy();
-        const input = shallow(<TagInput values={VALUES} inputProps={{ autoFocus: true, onBlur }} />).find("input");
+        const input = mount(<TagInput values={VALUES} inputProps={{ autoFocus: true, onBlur }} />).find("input");
         assert.isTrue(input.prop("autoFocus"));
         // check that event handler is proxied
         const fakeEvent = { flag: "yes" };
-        input.simulate("blur", fakeEvent);
+        input.prop("onBlur")?.(fakeEvent as any);
```

Comment:
I don't love this change but it appears to be the best path forward given the Enzyme issues you just linked. I'm surprised to learn that `.simulate()` is no longer recommended. And at some point I will move away from Enzyme anyway so these tests will get updated...

Reply:
agreed